fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3448,6 +3438,33 @@ public void cancelRefreshMaterializedView(String dbName, String mvName) throws D
         if (materializedView == null) {
             throw new MetaNotFoundException(mvName + " is not a materialized view");
         }
+        return materializedView;
+    }
+
+    @Override
+    public void refreshMaterializedView(RefreshMaterializedViewStatement refreshMaterializedViewStatement, int priority)
+            throws DdlException, MetaNotFoundException {
+        String dbName = refreshMaterializedViewStatement.getMvName().getDb();
+        String mvName = refreshMaterializedViewStatement.getMvName().getTbl();
+        boolean force = refreshMaterializedViewStatement.isForceRefresh();
+        RangePartitionWithoutIntervalDesc range =
+                refreshMaterializedViewStatement.getRangePartitionWithoutIntervalDesc();
+        if (range == null && !force) {
+            refreshMaterializedView(dbName, mvName, priority);
```

Comment:
I think you can unify the logic of if and else into the same function. It is ambiguous here

Reply:
yes, It is ambiguous here